seatunnel-connectors-v2/connector-pulsar/src/main/java/org/apache/seatunnel/connectors/seatunnel/pulsar/source/reader/PulsarSplitReaderThread.java

```diff
@@ -100,14 +100,13 @@ public void run() {
                 Thread.sleep(pollInterval);
             }
         } catch (Throwable t) {
-            handover.reportError(t);
-        } finally {
```

Comment:
Why remove the finally? Don't you need to ensure that the consumer is closed in finally?

Reply:
It is a bug. Refence #4706